modules/test/playwright/tests/analytics-web/cookiesPreferenceHandling.spec.ts

```diff
@@ -0,0 +1,192 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {featureFlagsTest} from '../../fixtures/featureFlagsTest';
+import {loginAnalyticsCloudTest} from '../../fixtures/loginAnalyticsCloudTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {liferayConfig} from '../../liferay.config';
+import {HomePage} from '../../pages/portal-web/HomePage';
+import {
+	connectToAnalyticsCloud,
+	disconnectFromAnalyticsCloud,
+	goToAnalyticsCloudInstanceSettings,
+	syncAllContacts,
+	syncSite,
+} from '../analytics-settings-web/utils/analyticsSettings';
+import {createDataSource} from '../osb-faro-web/utils/dataSource';
+
+export const test = mergeTests(
+	loginAnalyticsCloudTest(),
+	loginTest(),
+	featureFlagsTest({
+		'LPD-10588': true,
+	})
+);
+
+async function changeCookiePreferenceHandling(
+	page,
+	{enableCookie, enableExplicitCookieConsentMode}
+) {
+	const homepage = new HomePage(page);
+
+	await homepage.openApplicationMenu();
+
+	await page.getByRole('tab', {name: 'Control Panel'}).click();
+
+	await page.getByRole('menuitem', {name: 'Instance Settings'}).click();
+
+	await page.getByRole('link', {name: 'Cookies'}).click();
+
+	if (enableCookie) {
+		await page.getByLabel('Enabled').check();
+	} else {
+		await page.getByLabel('Enabled').uncheck();
+	}
+
+	if (enableExplicitCookieConsentMode) {
+		await page.getByLabel('Explicit Cookie Consent Mode').check();
+	} else {
+		await page.getByLabel('Explicit Cookie Consent Mode').uncheck();
+	}
+
+	await page.getByRole('button', {name: 'Update'}).click();
+
+	await page
+		.getByText('Success:Your request completed successfully.')
+		.click();
+}
+
+async function connectACWithDXP(page) {
+	await createDataSource(page);
+
+	await goToAnalyticsCloudInstanceSettings(page);
+
+	const cookieBannerElement = await page.$('div.portlet-cookies-banner');
+
+	if (cookieBannerElement) {
+		await cookieBannerElement.evaluate((div) => {
+			div.style.display = 'none';
+		});
+	}
+
+	await disconnectFromAnalyticsCloud(page);
+
+	await connectToAnalyticsCloud(page);
+
+	await syncSite(page);
+
+	await syncAllContacts(page);
+
+	await page.getByRole('button', {name: 'Finish'}).click();
+}
```

Comment:
Hey @interaminense, I think you can move this to the `analyticsSettings.ts` file, because this is a function that will be used in practically all AC tests and not something exclusive to the `cookiesPreferenceHandling.spec.ts` file. What do you think?

Reply:
It is also used on analyticsSettings.spec.ts test